app/controllers/v1/statuses_controller.rb

```diff
@@ -7,6 +7,15 @@ class StatusesController < ApplicationController
 
     ARB = ActiveRecord::Base
 
+    # TODO: Account policy for statuses?
+    before_action only: %i[index] do
+      Pundit.authorize(current_user, ::Account, :index?, policy_class: AccountPolicy)
+    end
+
+    before_action only: %i[show] do
+      Pundit.authorize(current_user, ::Account, :show?, policy_class: AccountPolicy)
+    end
+
```

Comment:
No need to put RBAC on the status, this is being consumed by the readiness/liveness probe.